vm_callinfo.h

```diff
@@ -12,12 +12,13 @@
 #include "internal/class.h"
 #include "shape.h"
 
-enum vm_call_flag_bits {
+enum vm_call_flag_bits
+{
```

Comment:
It seems like we don't do that style for things like structs/enums. Let's keep unrelated stuff as is.

```suggestion
enum vm_call_flag_bits {
```